src/api.rs

```diff
@@ -264,7 +264,13 @@ impl Context {
         } as u8) & 7;
       }
 
+      self.fi.reference_mode = if multiref && reorder && idx_in_group != 0 {
```

Comment:
Shouldn't just be "if multiref"?

Reply:
No. reference_mode == ReferenceMode::SELECT is synonymous to compound mode can be enabled at the block level. Multiref is synonymous to one of multiple reference frames can be selected at the block level